src/reuse/comment.py

```diff
@@ -392,6 +393,26 @@ class ModernFortranCommentStyle(CommentStyle):
     INDENT_AFTER_SINGLE = " "
 
 
+class ErlangCommentStyle(CommentStyle):
+    """Erlang comment style."""
+
+    SHORTHAND = "erlang"
+
+    SINGLE_LINE = "%"
+    INDENT_AFTER_SINGLE = " "
+    SHEBANGS = ["#!"]
+
+
+class ElixirCommentStyle(CommentStyle):
```

Comment:
This looks identical to `PythonCommentStyle`. Do you see any differences or do you think they styles will diverge in the future?

Reply:
I do not see any differences, but I thought that maybe one does not want to conflate comment styles for different languages, in case in a few years one comment style changes a bit, while the other should not.

Happy to be wrong and I can fix to use Python style, if you think this is more maintainable 😃

Reply:
I mean I'm just getting back into the codebase, but from what I can see, many languages use the same code style definition and it wouldn't be an issue to add another one should they diverge in the future.

Reply:
Ok, can you confirm that I understand that the preference is to follow `PythonCommentStyle`? 
Just to confirm that I understood correctly that I should modify this

Reply:
I indeed did not miss anything here and for identical comment styles, we use an existing one. So in this case, `PythonCommentStyle` is identical and we can go with that.